app/src/main/res/values/settings_keys.xml

```diff
@@ -55,6 +55,25 @@
         <item>30000</item>
     </string-array>
 
+    <string name="progressive_load_interval_key">progressive_load_interval</string>
+    <string name="progressive_load_interval_bytes_default_value">16384</string>
```

Comment:
That is not the default value according to https://github.com/google/ExoPlayer/blob/029a2b27cbdc27cf9d51d4a73ebeb503968849f6/library/core/src/main/java/com/google/android/exoplayer2/source/ProgressiveMediaSource.java#L256

I think adding an default entry that say's "use ExoPlayer's default value" (or something similar) would be great.
If this value (e.g. -1) is encountered the continueLoadingCheckIntervalBytes will be untoughed in the code.

Reply:
Setting the default to 16KiB was to help testers to see the effect right away before muddling with the config.
I've updated that to reflect the ExoPlayer default now, and yes, using the default value from lib is better than having it hardcoded on our side. Fixed.